drivers/overlay/overlay.go

```diff
@@ -631,6 +626,18 @@ func (d *Driver) create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 		}
 	}
 
+	perms := defaultPerms
+	if d.options.forceMask != nil {
```

Comment:
I think this should have precedence and override the parent mode

Reply:
Hmm, yeah, you're right.  Changing it.